composer.json

```diff
@@ -88,6 +88,7 @@
         "drupal/starshot_installer": "*",
         "drupal/starshot_multilingual": "*",
         "drush/drush": "^12.5",
+        "drupal-eca-recipe/notify_new_comments": "dev-main",
```

Comment:
Let's move this to `recipes/starshot_blog/composer.json`, since it's really going to be used by the starshot_blog recipe.

Reply:
I've moved it into that composer file.